### PR TITLE
Missing type annotations for EXT

### PR DESCRIPTION
**Pull request by OlivierBBB**

### CODE REVIEW

File: alu/ext/columns.lisp [hunk 1]
```diff
@@ -1,19 +1,19 @@
 (module ext)
 
 (defcolumns
-    STAMP
-    (OLI :binary)
-    CT
-    INST
+    (STAMP   :i64)
```
Comment (letypequividelespoubelles) [inline alu/ext/columns.lisp:4]: Why not :i32 as for all other stamps ?

File: blake2fmodexpdata/columns.lisp [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-  (IS_MODEXP_BASE :binary@prove)
-  (IS_MODEXP_EXPONENT :binary@prove)
-  (IS_MODEXP_MODULUS :binary@prove)
-  (IS_MODEXP_RESULT :binary@prove)
-  (IS_BLAKE_DATA :binary@prove)
-  (IS_BLAKE_PARAMS :binary@prove)
-  (IS_BLAKE_RESULT :binary@prove))
-
+  (STAMP                :byte)
+  (ID                   :i48)
```
Comment (letypequividelespoubelles) [inline blake2fmodexpdata/columns.lisp:5]: Hub_Stamp is the only stamp to be :i48 and not :i32 ?

**Review by letypequividelespoubelles — APPROVED** [review]
Please don't modify the formatting in the same PR, it makes review less understandable

File: alu/ext/columns.lisp (hunk 1 above)
Comment (OlivierBBB) [reply alu/ext/columns.lisp:4]: No reason, just a quick fix. `:i32` works fine, too.

File: blake2fmodexpdata/columns.lisp (hunk 2 above)
Comment (OlivierBBB) [reply blake2fmodexpdata/columns.lisp:5]: I think we can go either way for the ID. IIRC the STAMP counts without gaps. But even with current limitations (600 rounds in total for BLAKE if memory serves) that `:byte` is insufficient.

**Pull request merged**